en/log/flight_log_analysis.md

```diff
@@ -110,7 +110,7 @@ Key features:
 ![pyFlightAnalysis 1.0.1b1](../../assets/flight_log_analysis/pyflightanalysis.png)
 
 
-### FlightPlot (Desktop)
+### FlightPlot {#flightplot}
```

Comment:
FYI, did you mean to get rid of (Desktop) in the heading? Note that the good thing about the anchor syntax `{#flightplot}` is that it no longer matters what the preceding text in the heading is - the anchor stays the same.

Anyway, I don't think this breaks anything. Merging.

Reply:
Yes I deliberately got rid of `(Desktop)` because it looks ugly and most other tools listed are of the same type "for desktop computers" which was probably the meaning.